### PR TITLE
feat(FWS -37): Manage system analytics

### PR DESCRIPTION
**Pull request by emilegirardGit**
JIRA: link to jira ticket
https://champlainsaintlambert.atlassian.net/browse/FWS-38

Context:
This ticket was about adding information to the personal trainer and admin about the website.

Changes
-Added new user by month and year
-Added appointments total by status and by year
UI Changes
NEW
Analytics

### CODE REVIEW

File: fitwsarah_frontend/src/App.js [hunk 1]
```diff
@@ -50,6 +51,7 @@ function App() {
                     <Route path="/trainerFeedback" element={<Feedbacks/>}/>
                     <Route path="/adminFeedback" element={<AdminFeedback/>}/>
                     <Route path="/invoices" element={<ClientInvoices/>}/>
+                    <Route path="/analytics" element={<Analytics/>}/>
```
Comment (Eli293) [inline fitwsarah_frontend/src/App.js:54]: So will the trainer use the same analytics page as the admin?

**Review by Eli293 — APPROVED** [review]
Looks good.

**Review by AlexeiTimbro — APPROVED** [review]
looks good

File: fitwsarah_frontend/src/App.js (hunk 1 above)
Comment (emilegirardGit) [reply fitwsarah_frontend/src/App.js:54]: Its the same anyway so why would I just duplicate code

**Pull request merged**